components/eventing-controller/pkg/backend/nats/jetstream/jetstream.go

```diff
@@ -420,13 +420,18 @@ func (js *JetStream) syncSubscriptionFilters(subscription *eventingv1alpha1.Subs
 }
 
 func (js *JetStream) syncSubscriptionFilter(key SubscriptionSubjectIdentifier, subscription *eventingv1alpha1.Subscription, subscriber backendnats.Subscriber, log *zap.SugaredLogger) error {
```

Comment:
could we rename this function? If i understand it correctly, this function does not really sync the Subscription, but it cleans up consumers/subscriptions on nats from our internal cache